### PR TITLE
Minor edger8r and oeedl_file tweaks.

### PR DESCRIPTION
**Pull request by anakrish**
1. Enable --header-only option.
 2. Add --edl-search-dir option to oeedl_file.
 3. Lockdown selective import of functions in an edl file in search-dir.

### CODE REVIEW

**Comment by anakrish** [comment]
bors try

File: cmake/oeedl_file.cmake [hunk 1]
```diff
@@ -9,31 +9,53 @@
...
-#               <edl_file> <type> <out_files_var>
+#               <edl_file> <type> <out_files_var> [--edl-search-dir dir]
 #
 # Arguments:
 # edl_file - name of the EDL file
-# type - type of files to genreate ("enclave" or "host")
+# type - type of files to genreate ("enclave" or "host" or "enclave-headers" or "host-headers")
 # out_files_var - variable to get the generated files added to
-#
+# --edl-search-dir dir - Additional folder to look for imported edl files.
```
Comment (paulcallen) [inline cmake/oeedl_file.cmake]: "Additional folder **relative to the source directory** to look for imported edl files."

Also, does `oeedger8r --help` need updating as well, or does it pull it from here?

**Pull request merged**